02-getting_started.Rmd

```diff
@@ -1,27 +1,53 @@
-# Getting Started 
-
-## Install git
-
-## Get the source code 
-
-## Compile and build
-
-### UNIX
-
-### Windows
-
-## Install dependencies
-
-### Linux
-
-### maxOS and OS X
-
-## Regenerate `configure`
-
-## Troubleshoot the build
-
-### Avoid recreating auto-generated files
-
-## Editors and Tools
-
-## Directory structure
+# Getting Started 
+
+These instructions cover how to install $\textsf{R}$ in Windows. The tools required to build $\textsf{R}$ and $\textsf{R}$ packages in Windows are also discussed.
+
+## Installing $\textsf{R}$
+
+1. The binary builds of $\textsf{R}$ for Windows can be downloaded and installed from [here](https://cran.r-project.org/bin/windows/base/). Along with the link to the latest stable release, this page also contains links to the binary builds of r-patched and r-devel. 
+
+2. Click on the download links to download an executable installer.
+
+3. Select the language while installing, read the public license information, and select destination location to the start the installation. You will be prompted to select components at this stage: `User installation`, `32-bit User installation`, `64-bit User installation`, or `Custom installation`. The default option may be opted for the questions from this step onwards to complete the installation. 
+
+## General instructions
+
+1. If you install the latest version or R-patched or R-devel, it will not over-write the previous installation(s) in your Windows machine.
+
+2. R uses a ‘major.minor.patchlevel’ version numbering scheme. Accordingly there are three are different versions of $\textsf{R}$ available as binary builds: the latest official release (`r-release`), the latest patched release (`r-patched`), and the latest development (`r-devel`). The `r-devel` is the next minor or eventually major release development version of $\textsf{R}$. Mostly, bug fixes are introduced in `r-patched`, while `r-devel` is for introducing new features.

```

Comment:
I would move the `General instructions` to before `Installing R`. Since the versions of R also apply to source releases (not just binary builds). The second sentence can be changed to "Accordingly there are three main releases of R available to install". Then I would list them in bullet points to make it easier to read, with the last two sentences in a paragraph after the bullets. I think it was me that said "latest official release" etc, but we should not say "latest" in the definition, as the name refers to the type of release, not necessarily the latest version of that release.